html/browser/common.css

```diff
@@ -319,4 +319,10 @@ div.notice
 {
 	width: 100%;
 	clear: both;
+}
+
+.charPreview{
+	-ms-interpolation-mode: nearest-neighbor; 
```

Comment:
honestly, you should just force this on all images.

and add the other versions of this css tag for the web client